### PR TITLE
removed theorem numbers

### PR DESCRIPTION
**Pull request by rubengamboa**
just testing pull request procedure

### CODE REVIEW

**Review by rexpage — APPROVED** [review]
It's okay with me to go ahead with this. Not sure how it's done. I think you revised the theorem tag, rather than removing it. Is that right?

File: book.tex [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 %\usepackage{amsmath}
 \usepackage{amsthm}
-\newtheorem{theorem}{Theorem}
+\newtheorem*{theorem}{Theorem}
```
Comment (rexpage) [inline book.tex:20]: Is the asterisk the only difference? Does this remove theorem number somehow? Does it leave the rest of the theorem attributes in place?

**Comment by rubengamboa** [comment]
Yep, that removes the theorem number but leaves everything else.

Clear skies,
Ruben

--
Ruben Gamboa
Professor
Dept. of Computer Science
University of Wyoming

**Comment by rexpage** [comment]
I clicked the merge button, and github responded with merged commit 4d4a06c. I guess it's there. Not sure how I download it.

**Comment by rexpage** [comment]
Downloaded zip with everything, copied book.tex to my local folder, and it seems to work. No theorem numbers.

**Comment by rubengamboa** [comment]
Yes!

Clear skies,
Ruben

--
Ruben Gamboa
Professor
Dept. of Computer Science
University of Wyoming

**Comment by rexpage** [comment]
Downloaded zip with everything, and I must have gotten the change because
theorems aren't numbered now.

**Pull request merged**